### DISCUSSION TITLE
Any New Tab / Homepage Add-Ons compatible with Vimium?

### DISCUSSION THREAD

**Discussion by J-Kappes** [philc/vimium#4393] [General]
I love the Vomnibar, but it doesn't work on Firefox's homepage, Tabliss or any other homepage Add-On I've tried :(

Any suggestions? Support for bookmarks and recently visited Sites would be nice to have.

**Comment by CallMePixelMan** [comment]
By any chance, did you find a workaround? I'm facing the same issue. Thanks!

**Reply by J-Kappes** [reply]
No, I haven't :( I just use the URL bar or open a random site first.

**Reply by J-Kappes** [reply]
I'm now using [web.tabliss.io ](https://github.com/joelshepherd/tabliss) as a homepage.

**Reply by seki-mei** [reply]
I use [New Tab Override](https://addons.mozilla.org/de/firefox/addon/new-tab-override/) set to `moz-extension://<extension-id>/pages/blank.html`. You can find the page by opening the settings page for vimium and replacing `/pages/options.html` with `/pages/blank.html`.  
 It's a local page and Vimium works there.  

The only issue I have with this workaround is that the New Tab Override plugin has an annoying quirk where if you close a tab, open a new tab, and then try to restore the closed tab, firefox opens a new tab instead. Any further `ctrl-shift-T`s will just keep creating new tabs. Maybe New Tab Override closes firefox's default new tab before opening its own? At any rate, this isn't a big deal because usually when I need to restore a tab it's immediately after I close it. And it's also faster to use `T` or `B` to launch pages or searches in a new tab.
